### PR TITLE
Rename Bitreserve to Uphold

### PR DESCRIPTION
**Pull request by JoaoHenriquePereira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Bitreserve API Reference
+# Uphold API Reference
 
-This repository contains the source files of the Bitreserve API documentation which can be viewed at https://developer.bitreserve.org.
+This repository contains the source files of the Uphold API documentation which can be viewed at https://developer.uphold.com.
```
Comment (ruimarinho) [inline README.md]: `https://uphold.com/en/developer/api/documentation`

File: _applications.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Developers will need to [register their application](https://bitreserve.org/dash
 * For security reasons, your application **must** be secured with a valid *SSL* certificate issued by a known Certificate Authority.
 * Likewise, the provided *Redirect URL* when registering the application must be a valid static subresource. Notice that this property cannot be dynamically reconfigured during authorization requests for security reasons.
 * Users can revoke access to your application at any time. Your application **must** be prepared for this and, if necessary, should request authorization from the user again.
-* Your application may be suspended in an automated fashion in accordance with our [Terms of Service](https://bitreserve.org/en/tos).
+* Your application may be suspended in an automated fashion in accordance with our [Terms of Service](https://uphold.com/en/tos).
```
Comment (ruimarinho) [inline _applications.md]: `https://uphold.com/en/legal/membership-agreement`

File: _applications.md [hunk 3]
```diff
@@ -40,22 +40,49 @@ You can always request more scopes later by asking for user consent again.
 
 ## Resources
 
-We prefer that you use these image resources when connecting your applications to Bitreserve.
+We prefer that you use these image resources when connecting your applications to Uphold.
 
-<img alt="Connect Button" src="images/buttons/bitreserve-connect-light-purple@1x.png" srcset="images/buttons/bitreserve-connect-light-purple@1x.png 1x, images/buttons/bitreserve-connect-light-purple@2x.png 2x"><br>
-[small (119x40)](images/buttons/bitreserve-connect-light-purple@1x.png), [large (238x80)](images/buttons/bitreserve-connect-light-purple@2x.png)
+<img alt="Connect Button" src="images/buttons/greenBg_btn@1x.png" srcset="images/buttons/greenBg_btn@1x.png 1x, images/buttons/greenBg_btn@2x.png 2x"><br>
```
Comment (ruimarinho) [inline _applications.md]: @diogobarm is renaming these files in his PR.

File: _authentication.md [hunk 4]
```diff
@@ -95,9 +95,9 @@ For **personal usage only** you may choose to use a PAT. This token establishes
 > To create a Personal Access Token, execute the following command:
 
 ```bash
-curl https://api.bitreserve.org/v0/me/tokens \
+curl https://api.uphold.com/v0/me/tokens \
   -X POST \
-  -H 'X-Bitreserve-OTP: <OTP-Token>' \
+  -H 'X-Uphold-OTP: <OTP-Token>' \
```
Comment (ruimarinho) [inline _authentication.md]: This rename hasn't yet occurred, so we should postpone and ticket this.

File: _authentication.md [hunk 5]
```diff
@@ -116,7 +116,7 @@ description | no | A human-readable description of this PAT.
 <aside class="notice">
 **Import Notice**: This request must be authenticated with your username and password using the HTTP Basic Authentication scheme.
 
-Additionally, if the account is secured with Two-Factor Authentication, the `X-Bitreserve-OTP` header is required.
+Additionally, if the account is secured with Two-Factor Authentication, the `X-Uphold-OTP` header is required.
```
Comment (ruimarinho) [inline _authentication.md]: Same as above.

File: _authentication.md [hunk 6]
```diff
@@ -157,11 +157,11 @@ The username should be set as the `token` and password should be set to `X-OAuth
 > Simple request using username or email and password:
 
 ```bash
-curl https://api.bitreserve.org/v0/me \
-  -H 'X-Bitreserve-OTP: <OTP-Token>' \
+curl https://api.uphold.com/v0/me \
+  -H 'X-Uphold-OTP: <OTP-Token>' \
```
Comment (ruimarinho) [inline _authentication.md]: Same as above.

File: _authentication.md [hunk 7]
```diff
@@ -157,11 +157,11 @@ The username should be set as the `token` and password should be set to `X-OAuth
...
   -u <username-or-email>:<password>
 ```
 
 You can use Basic Authentication by providing your username or email and password combination.
 
-If you have OTP (One Time Password, also known as Two-Factor Authentication) enabled, then you will get an HTTP 401 (Unauthorized) response, along with the HTTP header `X-Bitreserve-OTP: Required`. You will then automatically receive an SMS, or Push Notification with your verification code, depending on whether you have the Authy app installed or not. Then execute the command above again, this time passing your OTP verification code as a header, like so: `X-Bitreserve-OTP: <OTP-Token>`.
+If you have OTP (One Time Password, also known as Two-Factor Authentication) enabled, then you will get an HTTP 401 (Unauthorized) response, along with the HTTP header `X-Uphold-OTP: Required`. You will then automatically receive an SMS, or Push Notification with your verification code, depending on whether you have the Authy app installed or not. Then execute the command above again, this time passing your OTP verification code as a header, like so: `X-Uphold-OTP: <OTP-Token>`.
```
Comment (ruimarinho) [inline _authentication.md]: Same as above.

File: _currencies.md [hunk 8]
```diff
@@ -1,4 +1,5 @@
 # Currencies
 
-In our API, we frequently call for a currency as an input or output. We represent all such currencies and stores of value by a three letter code. In our Knowledge Base you will always find the most recent and up-to-date [list of the currencies we support](https://support.bitreserve.org/hc/en-us/articles/202473803), along with the three letter abbreviation you can use when referring to the currency within our API. 
+In Uphold's API, we frequently call for a currency as an input or output. We represent all such currencies and stores of value by a three letter code. Listed below are the currencies and stores of value that we currently support.
```
Comment (ruimarinho) [inline _currencies.md]: Please mention that we're using "ISO 4217 codes".

File: _authentication.md [hunk 9]
```diff
@@ -105,7 +105,7 @@ curl https://api.bitreserve.org/v0/me/tokens \
 
 To create a Personal Access Token you may use the following endpoint:
 
-`POST https://bitreserve.org/v0/me/tokens`
+`POST https://uphold.com/v0/me/tokens`
```
Comment (fixe) [inline _authentication.md]: api.uphold.com

File: _authentication.md [hunk 10]
```diff
@@ -124,14 +124,14 @@ Additionally, if the account is secured with Two-Factor Authentication, the `X-B
...
-curl https://api.bitreserve.org/v0/me/tokens/:token \
+curl https://api.uphold.com/v0/me/tokens/:token \
   -X DELETE \
   -H "Authorization: Bearer <token>"
 ```
 
 To revoke a Personal Access Token you may use the following endpoint:
 
-`DELETE https://bitreserve.org/v0/me/tokens/:token`
+`DELETE https://uphold.com/v0/me/tokens/:token`
```
Comment (fixe) [inline _authentication.md]: api.uphold.com

File: _introduction.md [hunk 11]
```diff
@@ -1,21 +1,21 @@
...
 * [Ruby SDK](https://github.com/groupbuddies/bitreserve)
 
-We are actively looking for developers to contribute Bitreserve client libraries for a variety of platforms and languages. If you have written a library, please share a link to the github repository with us at <a href="mailto:developer@bitreserve.org?subject=I want to contribute code">developer@bitreserve.org</a> so that we can promote it here, and collaborate with you on its further development.
+We are actively looking for developers to contribute Uphold client libraries for a variety of platforms and languages. If you have written a library, please share a link to the github repository with us at <a href="mailto:developer@uphold.com?subject=I want to contribute code">developer@uphold.com</a> so that we can promote it here, and collaborate with you on its further development.
 
 ## API Changes
 
-You can be notified of the latest changes by *watching* our official [documentation repository](https://github.com/bitreserve/api/) on GitHub.
+You can be notified of the latest changes by *watching* our official [documentation repository](https://github.com/uphold/api/) on GitHub.
```
Comment (fixe) [inline _introduction.md]: This should be done in another PR since the organization rename hasn't been made yet.

File: _introduction.md [hunk 12]
```diff
@@ -1,21 +1,21 @@
...
 
-We are currently in *v0* which means that breaking changes may be introduced. If there are any upcoming changes we will add them to a release named *next*. You can follow it in the *[releases](https://github.com/bitreserve/api/releases)* page to avoid breaking your application.
+We are currently in *v0* which means that breaking changes may be introduced. If there are any upcoming changes we will add them to a release named *next*. You can follow it in the *[releases](https://github.com/uphold/api/releases)* page to avoid breaking your application.
```
Comment (fixe) [inline _introduction.md]: Same as above.

File: _introduction.md [hunk 13]
```diff
@@ -1,21 +1,21 @@
...
 
-The official changelog is currently on [GitHub](https://github.com/bitreserve/api/releases).
```
Comment (fixe) [inline _introduction.md]: Same as above.
